akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1058,6 +1059,33 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
     new Flow(delegate.mergeMat(that)(combinerToScala(matF)))
 
   /**
+   * Merge the given [[Source]] to this [[Flow]], taking elements as they arrive from input streams,
+   * picking always the smallest of the available elements (waiting for one element from each side
+   * to be available).
+   *
+   * '''Emits when''' one of the inputs has an element available
+   *
+   * '''Backpressures when''' downstream backpressures
+   *
+   * '''Completes when''' all upstreams complete
+   *
+   * '''Cancels when''' downstream cancels
+   */
+  def mergeSorted[U >: Out, M](that: Graph[SourceShape[U], M], comp: Comparator[U]): javadsl.Flow[In, U, Mat] =
```

Comment:
I actually quite like the name because it matches “merge sorted streams” as well as that the result of that operation is a sorted stream (i.e. if the inputs were sorted).